modules/core/test/unit/externalApi.ts

```diff
@@ -0,0 +1,112 @@
+const Wallet = require('../../src/wallet');
+import { TestBitGo } from '../lib/test_bitgo';
+import * as _ from 'lodash';
+import * as Bluebird from 'bluebird';
+const co = Bluebird.coroutine;
+import * as nock from 'nock';
+import * as common from '../../src/common';
+
+nock.disableNetConnect();
+const externalHost = 'https://externalsign.com';
```

Comment:
please use an invalid TLD for testing, as per https://datatracker.ietf.org/doc/html/rfc2606#page-2

I prefer `.invalid`